python/feathub/feature_views/feature_view.py

```diff
@@ -133,25 +133,28 @@ def get_output_fields(self, source_fields: List[str]) -> List[str]:
             raise FeathubException(
                 "Build this feature view before getting output fields."
             )
-        output_fields = []
+
+        derived_fields = [feature.name for feature in self.get_resolved_features()]
```

Comment:
How about moving this part of logic to `DerivedFeatureView`? `derived_fields` only exists in `DerivedFeatureView` and different views can have different ordering strategies.

Reply:
It is called derived_fields but not only used in `DerivedFeatureView`, I renamed it with `feature_fields`.